nix/haskell-extra.nix

```diff
@@ -58,31 +58,31 @@ in {
     version = "2.2.11";
     inherit compiler-nix-name index-state checkMaterialization;
     # Invalidate and update if you change the version or index-state
-    plan-sha256 = "10zw1zvs2bbdyh2mg01vm8wz6n65m2qkvz1va9hzhj8ch00nvpbh";
+    plan-sha256 = "0mw6w1b79hww4m5r8wncgbyw03p34mmmm66rl8wglq3ynjxfxh77";
   };
-  inherit (
-    let hspkgs = pkgs.haskell-nix.cabalProject {
+  inherit (pkgs.haskell-nix.cabalProject {
         src = pkgs.fetchFromGitHub {
+          name = "haskell-language-server";
           owner = "haskell";
           repo = "haskell-language-server";
-          rev = "2186df00a9414c640fba1ae2acc3d9aa21ab6e4c";
-          sha256 = "0qh41jbf1a697l8wf48zmfs6vf08gijb0w42h26nvimcgc5dkh9a";
+          rev = "15f870f89a18ca3e7991193f8c996ac6b4e17b26";
+          sha256 = "14r6s1yw4h8b6jc4v04zk7hinpw23vn8jfk6sbbq7798nw891k7g";
           fetchSubmodules = true;
         };
-        lookupSha256 = { location, tag, ... } : {
-          "https://github.com/wz1000/shake"."fb3859dca2e54d1bbb2c873e68ed225fa179fbef" = "0sa0jiwgyvjsmjwpfcpvzg2p7277aa0dgra1mm6afh2rfnjphz8z";
-          "https://github.com/peti/cabal-plan"."894b76c0b6bf8f7d2f881431df1f13959a8fce87" = "06iklj51d9kh9bhc42lrayypcpgkjrjvna59w920ln41rskhjr4y";
-          }."${location}"."${tag}";
+        sha256map = {
+          "https://github.com/bubba/brittany.git"."c59655f10d5ad295c2481537fc8abf0a297d9d1c" = "1rkk09f8750qykrmkqfqbh44dbx1p8aq1caznxxlw8zqfvx39cxl";
+        };
         inherit compiler-nix-name index-state checkMaterialization;
         # Invalidate and update if you change the version
-        plan-sha256 = "0a6c4lhnlm2lkic91ips0gb3hqlp3fk2aa01nsa8dhz9l8zg63da";
+        plan-sha256 = if pkgs.hostPlatform.isLinux
+          then "0jq1a2iyn394s3d2xag45d8ga32gn1i5bn5i63xd1jqllb85pcw3"
+          else "0l812savqj4xch9afky93zfkn5wvq6mz3q09v6mxkhcpnp6gshic";
```

Comment:
yikes

Reply:
I think I might also have seen this... this seems very bad. `nix-tools` bug?

Reply:
Calculated plan is different as a result of `if os( darwin )` in cabal files.